YvesIraguha/TeaHouse
# #169725605 Pagination Stories/Poems

#### What does this PR do?
- Enable paginations and filtering on Stories/Poems
#### Description of Task to be completed?
- Paginate the output of fetching all articles
- Filter the output based on stories or poems
#### How should this be manually tested?
- Clone the repo and checkout the branch `pagination-stories-poems-169725605`
- Run `npm i && npm run precommit && npm run start` to run tests and start the server
- Consume the endpoint `/api/v1/individual-pieces?page=number&type=Poem`, and you should see the paginated outcome.

#### What are the relevant pivotal tracker stories?
#[169725605](https://www.pivotaltracker.com/story/show/169725605)
